.github/workflows/build.yml

```diff
@@ -37,6 +37,10 @@ jobs:
         pip install -r windows-build\windows_build_requirements.txt
     - name: Build app
       run: |
+        cd src/main/resources/base/ayab/translations
+        ./ayab_trans.pl
+        lrelease *.ts
+        cd ../../../../../..
```

Comment:
Should be possible to simply cd back to $HOME (or ~) (see https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables)

Reply:
OK. I made a new workflow step to create the translation files. I'm not familiar with these workflows, so I hope that works.